libs/blocks/table/table.js

```diff
@@ -575,22 +595,44 @@ function applyStylesBasedOnScreenSize(table, originTable) {
     }
   };
 
+  const removeClones = () => {
+    const rows = table.querySelectorAll('.row');
+
+    rows.forEach((row) => {
+      const clonedCols = row.querySelectorAll('.col[data-cloned]');
+      clonedCols.forEach((col) => col.remove());
+    });
+  };
+
   // For Mobile (else: tablet / desktop)
   if (!isMerch && !table.querySelector('.row-heading .col-2')) {
     table.querySelector('.row-heading').style.display = 'block';
     table.querySelector('.row-heading .col-1').style.display = 'flex';
   }
 
   if (deviceBySize === 'MOBILE' || (isMerch && deviceBySize === 'TABLET')) {
+    removeClones();
     mobileRenderer();
   } else {
-    table.innerHTML = originTable.innerHTML;
-    reAssignEvents(table);
+    removeClones();
+    table.querySelectorAll('.hide-mobile').forEach((col) => { col.classList.remove('hide-mobile'); });
+    const element = table.querySelector('.row-heading');
+
+    if (element) {
+      const columns = [...element.children];
+      columns.forEach(({ children }) => {
+        [...children].forEach((row) => {
+          row.style.removeProperty('height');
+        });
+      });
+    }
```

Comment:
You should be able to avoid the if clause and shorten this a little
```suggestion
    [...(table.querySelector('.row-heading') || [])]
    .forEach({children}) => [...children].forEach(row => row.style.removeProperty('height'))
```